src/commands/register.ts

```diff
@@ -240,7 +240,8 @@ export default class ComponentRegister extends BaseCommand {
         });
       } catch (err: any) {
         fs.removeSync(compose_file);
-        this.error(err);
+        console.log(chalk.red(err));
+        process.exit(err.exitCode || 1);
```

Comment:
I don't know if it's ever possible for us to not get an exitCode from the execa exception, but technically this err is an `any` so I wanted to be safe

Reply:
Should we support some kind of:
`err.track = false`